src/home/pam_systemd_home.c

```diff
@@ -290,7 +295,7 @@ static int handle_generic_user_record_error(
         } else if (sd_bus_error_has_name(error, BUS_ERROR_TOKEN_PIN_NEEDED)) {
                 _cleanup_(erase_and_freep) char *newp = NULL;
 
-                r = pam_prompt(handle, PAM_PROMPT_ECHO_OFF, &newp, "Please enter security token PIN: ");
+                r = pam_prompt(handle, PAM_PROMPT_ECHO_OFF, &newp, "Please enter security PIN: ");
```

Comment:
hmm, but it's really the PIN of the security token, i.e. the fido2/pkcs#11 device, I think we should emphasize that, and hence say "security token".